.ci/build

```
@@ -94,55 +71,9 @@ if [[ $BRANCH != "master" && ! -z $BRANCH ]]; then
   (cd $CLONE && git checkout origin/${BRANCH})
 fi
 
-# Copy website contents like blogs adopters etc. from desired branch
-mkdir $HUGO_CONTENT
-cp -r $CLONED_WEBSITE_DIR $HUGO_CONTENT
-
-if [[ -d $HUGO_DOCUMENTATION ]]; then
-  rm -rf $HUGO_DOCUMENTATION
-fi
-
-if [[ "$BUILDSINGLEBRANCH" = "true" ]]; then
-  # Copy desired doc into hugo/content/documentation
-  cp -r $CLONED_DOCUMENTATION_PATH $HUGO_DOCUMENTATION
-
-  # Run docforge to download external files
-  docforge -f ${CLONE}/documentation.yaml -d $HUGO_DOCUMENTATION --resources-download-path __resources --hugo --github-oauth-token $GIT_OAUTH_TOKEN
-
-  export REMOTE=$HUGO_CONTENT
-  export CONTENT=CLONE/website
-  export DATA=hugo/data
-  node ./node/index.js
-else
-  echo "Building site from the last ${n} documentation versions"
-  n=${RELEASES_COUNT}
-  if [[ ${#LATESTVERSIONS[@]} -lt $n ]]; then
-    n=${#LATESTVERSIONS[@]}
-    echo "Releases count $n"
-  fi
-  for ((r = 0; r < n; r++)); do
-    version=${LATESTVERSIONS[$r]}
-    echo "Getting docs from: ${version}"
-    # Switch to desired release version
-    (cd $CLONE && git checkout "tags/${version}")
-
-    export REMOTE=$HUGO_CONTENT
-    export CONTENT=$CLONE/website/documentation
-    export DATA=hugo/data
-
-    # In case the latest documentation version set as root.
-    if [ $r = 0 ]; then
-      version="documentation"
-      export CONTENT=$CLONE/website
-    fi
-
-    cp -r $CLONED_DOCUMENTATION_PATH $HUGO_CONTENT/$version
-    docforge -f ${CLONE}/documentation.yaml -d $HUGO_CONTENT/$version --hugo --github-oauth-token $GIT_OAUTH_TOKEN
-    node ./node/index.js
-  done
-  node ./node/generateVersioningFile.js
-fi
-
+docforge -f ${CLONE}/.docforge/website.yaml -d $HUGO_CONTENT --resources-download-path __resources --hugo --github-oauth-token $GIT_OAUTH_TOKEN --github-info-destination ../data || true
```

The `__resources` value is the default for the `--resources-download-path ` flag so not strictly necessary here. 
```suggestion
docforge -f ${CLONE}/.docforge/website.yaml -d $HUGO_CONTENT --hugo --github-oauth-token $GIT_OAUTH_TOKEN --github-info-destination ../data || true
```